modules/querying-kubelet-status-on-a-node.adoc

```diff
@@ -0,0 +1,46 @@
+// Module included in the following assemblies:
+//
+// * support/verifying-node-health.adoc
+
+[id="querying-kubelet-status-on-a-node_{context}"]
+= Querying kubelet status on a node
+
+Review cluster node health status, resource consumption statistics and node logs. Additionally, query `kubelet` status on individual nodes.
+
+.Prerequisites
+
+* Access to the cluster as a user with the `cluster-admin` role.
+* Your API service is still functional.
+* Install the OpenShift Command-line Interface (CLI), commonly known as `oc`.
+
+.Procedure
+
+. kubelet is managed using a systemd service on each node. Review kubelet status by querying the `kubelet` systemd service within a debug Pod:
```

Comment:
The kubelet

Reply:
Thanks Andrea. I have now resolved this throughout the PR, where appropriate.